app/views/ExposureHistory/DetailedHistory.js

```diff
@@ -14,60 +19,94 @@ import { SingleExposureDetail } from './SingleExposureDetail';
  */
 export const DetailedHistory = ({ history }) => {
   const { detailedHistoryPageWhatThisMeansPara } = useAssets();
+  const { hasBeenExposed } = useContext(ExposureNotificationContext);
   const exposedDays = history.filter((day) => day.exposureMinutes > 0);
 
-  return (
-    <>
-      <ExposureCalendarView weeks={3} history={history} />
+  const Divider = () => {
+    return <View style={styles.divider} />;
+  };
 
-      <Divider />
+  const gpsExposureInfo = () => {
+    return (
+      <>
+        {exposedDays.map(({ exposureMinutes, date }) => (
```

Comment:
🍎 's `EN` api calls this property `duration` - should we do the same for consistency? 🤔

Reply:
@mattThousand , we definitely should, but lets grab that in a future commit.